docs/pallas/gpu/pipelining.md

```diff
@@ -0,0 +1,325 @@
+---
```

Comment:
Just OOC, is there any benefit for us in keeping this guild in both ipynb and as well .md? The only thing I can think of is that people might be able to run it on colab, but if it's mostly an explanation with not that much self-contained code then I'm not sure if it's useful?

Reply:
I'm fine with just keeping the md, but it's also not really any extra work to have the ipynb and there's a couple runnable examples here for the matmuls.